crates/pallet-messenger/src/lib.rs

```diff
@@ -76,6 +75,14 @@ pub struct InitiateChannelParams {
     max_outgoing_messages: u32,
 }
 
+#[derive(Debug, Encode, Decode, Clone, Eq, PartialEq, TypeInfo, Copy)]
+pub enum OutboxMessageStatus {
```

Comment:
I'd call it `OutboxMessageResult` as it does look like a Result type, plus there is the same `OutboxMessageStatus` variant in the Event.